executor/ddl.go

```diff
@@ -118,7 +114,14 @@ func (e *DDLExec) Next(ctx context.Context, req *chunk.Chunk) (err error) {
 
 	}
 	if err != nil {
-		return e.toErr(err)
+		// If owner return ErrTableNotExists error when running this DDL, It maybe cause by schema changed,
```

Comment:
If the owner return `ErrTableNotExists` error when running this DDL, it may be caused by schema changed,